radix-engine/src/model/validated_package.rs

```diff
@@ -29,47 +29,18 @@ pub enum PackageError {
 impl ValidatedPackage {
     /// Validates and creates a package
     pub fn new(package: scrypto::prelude::Package) -> Result<Self, WasmValidationError> {
-        let code = package.code().to_vec();
+        // TODO: validate should be a function rather than method
```

Comment:
Slightly disagree. Wasm engine is going to use stateful `cache` as in the other PR :)